spec/account_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+
+##note to self: class methods need module::class and instance methods need lowercaseclass.methodname
+describe "account attributes" do
+  context "responds to the account class" do
+    it "responds to the function new for Account" do
+      expect(BankAccount::Account).to respond_to :new
+    end
+  end
+
+  context "the user cannot open an account with a negative balance" do
+    it "raises an error" do
+      expect{BankAccount::Account.new(1,-5)}.to raise_error(ArgumentError)
+    end
+  end
+end
+
+describe "instance methods" do
+  let(:account) { BankAccount::Account.new(1,250)}
+
+  context "withdraw method removes money from account and returns updated balance" do
+    it "returns an updated balance of 45.00" do
```

Comment:
Since your test itself is what is checking for a specific balance, I would be less inclined to include that in your `it` descriptor.